app/controllers/concerns/pagination_concern.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module PaginationConcern
+  def extract_pagination_props
+    {
+      page: extract_int(params, :page),
+      limit: extract_int(params, :limit, 10)
```

Comment:
You can provide the default like this
```suggestion
      limit: extract_int(params, :limit) || 10
```
Let the `extract_int` return `nil` if there was no number provided